moxin-frontend/src/chat/chat_panel.rs

```diff
@@ -323,6 +366,9 @@ live_design! {
 
                 flow: Overlay
                 chat = <PortalList> {
+                    scroll_bar: {
+                        bar_size: 0.0,
```

Comment:
The whole portal list showing messages has a scroll bar that was not noticeable when everything was white, but know that chat bubbles have borders and the user bubble is not white, it is very noticeable. This hides the scroll bar again.